### PR TITLE
feat: update entity list item to include entity type [SPA-914]

### PR DESCRIPTION
**Pull request by ChidinmaOrajiaku**
# Purpose of PR
[Update entity list item to include entity type on card](https://contentful.atlassian.net/browse/SPA-914)

Screenshot 2022-11-17 at 14 46 49

## PR Checklist

- [ ] I have read the relevant `readme.md` file(s)
- [ ] All commits follow our [Git commit message convention](https://github.com/contentful/forma-36/tree/main/packages/forma-36-react-components#commits)
- [ ] Tests are added/updated/not required
- [ ] Tests are passing
- [ ] Storybook stories are added/updated/not required
- [ ] Usage notes are added/updated/not required
- [ ] Has been tested based on [Contentful's browser support](https://www.contentful.com/faq/about-contentful/#which-browsers-does-contentful-support)
- [ ] Doesn't contain any sensitive information

### CODE REVIEW

File: packages/components/entity-list/src/EntityListItem/EntityListItem.tsx [hunk 1]
```diff
@@ -219,17 +221,33 @@ export const EntityListItem = ({
...
-              >
-                {description}
+            <Flex marginTop="spacing2Xs">
+              {entityType && (
+                <Text
+                  as="span"
+                  lineHeight="lineHeightM"
+                  fontColor="gray600"
+                  className={styles.entityType}
+                  isTruncated
```
Comment (denkristoffer) [inline packages/components/entity-list/src/EntityListItem/EntityListItem.tsx:231]:
```suggestion
```

File: packages/components/entity-list/src/EntityListItem/EntityListItem.tsx [hunk 2]
```diff
@@ -219,17 +221,33 @@ export const EntityListItem = ({
...
-                lineHeight="lineHeightM"
-                fontColor="gray900"
-                isTruncated
-                className={styles.description}
-              >
-                {description}
+            <Flex marginTop="spacing2Xs">
+              {entityType && (
+                <Text
+                  as="span"
```
Comment (denkristoffer) [inline packages/components/entity-list/src/EntityListItem/EntityListItem.tsx:227]:
```suggestion
```

File: packages/components/entity-list/src/EntityListItem/EntityListItem.tsx (hunk 1 above)
Comment (denkristoffer) [reply packages/components/entity-list/src/EntityListItem/EntityListItem.tsx:231]: I'm thinking the entity type should always be fully shown, the alternative might look weird.

Comment (ChidinmaOrajiaku) [reply packages/components/entity-list/src/EntityListItem/EntityListItem.tsx]: Copy pasta mistake 😂

File: packages/components/entity-list/src/EntityListItem/EntityListItem.tsx (hunk 2 above)
Comment (ChidinmaOrajiaku) [reply packages/components/entity-list/src/EntityListItem/EntityListItem.tsx]: Changed

File: packages/components/entity-list/src/EntityListItem/EntityListItem.tsx [hunk 3]
```diff
@@ -219,17 +221,33 @@ export const EntityListItem = ({
...
+                  lineHeight="lineHeightM"
+                  fontColor="gray600"
+                  className={styles.entityType}
+                  isTruncated
+                >
+                  {entityType}
+                </Text>
+              )}
+              <Text as="span" lineHeight="lineHeightM" fontColor="gray400">
+                |
```
[the remark this answers is not in the document]
Comment (ChidinmaOrajiaku) [reply packages/components/entity-list/src/EntityListItem/EntityListItem.tsx]: Updated

**Review by denkristoffer — APPROVED** [review]
Thank you! 🙇

File: .changeset/witty-pants-stare.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+---
+'@contentful/f36-entity-list': major
```
Comment (massao) [inline .changeset/witty-pants-stare.md:2]: This isn't a major change, major would create a v5 release

**Review by massao — CHANGES_REQUESTED** [review]

**Review by Sarah-Roediger — APPROVED** [review]

**Comment by ChidinmaOrajiaku** [comment]
@massao Fixed

**Review by massao — APPROVED** [review]

**Pull request merged**